Openshift-EE/pipelines/GitLab/stages/4-chaos/N2QR-cstor-pool-delete/cstor-pool-delete

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#!/bin/bash
+set -x
+
+pod() {
+sshpass -p $pass_e2e ssh -o StrictHostKeyChecking=no $user_e2e@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/4-chaos/N2QR-cstor-pool-delete/cstor-pool-delete node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+source ~/.profile
+gittoken=$(echo "$github_token")
+case_id=N2QR
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash Openshift-EE/utils/e2e-cr jobname:cstor_pool_delete jobphase:Waiting
+bash Openshift-EE/utils/e2e-cr jobname:cstor_pool_delete jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+echo "*******Performing Cstor pool delete**********"
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=openebs-pool-failure metadata="")
```

Comment:
@shashank855  change the test case name. Use run_id to differentiate between cstor pool kill and cstor pool delete test case.